GitHubAction/GitHubAction.Console/Properties/launchSettings.json

```diff
@@ -2,7 +2,10 @@
   "profiles": {
```

Comment:
Although this is useful for debugging purposes. This file should not be included in git and is only used for local testing